Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/types.go

```diff
@@ -2523,6 +2526,12 @@ type SecurityContextConstraints struct {
 	SupplementalGroups SupplementalGroupsStrategyOptions
 	// FSGroup is the strategy that will dictate what fs group is used by the SecurityContext.
 	FSGroup FSGroupStrategyOptions
+	// ReadOnlyRootFilesystem when set to true will force containers to run with a read only root file
+	// system.  If the container specifically requests to run with a non-read only root file system
+	// the SCC should deny the pod.
+	// If set to false the container may run with a read only root file system if it wishes but it
+	// will not be forced to.
+	ReadOnlyRootFilesystem bool
```

Comment:
No.  This switch doesn't attempt to prevent it from being used.  When set to false it will allow you to use a ROFS but won't require it.  When set to true it will require it by either defaulting the unset value or rejecting a false value during admission